### PR TITLE
MM-37579: Fix typo in map property

### PR DESCRIPTION
**Pull request by agnivade**
https://mattermost.atlassian.net/browse/MM-37579

```release-note
NONE
```

### CODE REVIEW

File: app/email/email_test.go [hunk 1]
```diff
@@ -40,34 +40,71 @@ func TestSendInviteEmails(t *testing.T) {
...
-	mail.DeleteMailBox(emailTo)
+		var resultsMailbox mail.JSONMessageHeaderInbucket
+		err2 := mail.RetryInbucket(5, func() error {
+			var err error
+			resultsMailbox, err = mail.GetMailBox(emailTo)
+			return err
+		})
+		if err2 != nil {
+			t.Log(err2)
+			t.Log("No email was received, maybe due load on the server. Skipping this verification")
```
Comment (isacikgoz) [inline app/email/email_test.go:60]: nit: I wonder if should've use `t.Skip` here?

File: app/email/email_test.go [hunk 2]
```diff
@@ -40,34 +40,71 @@ func TestSendInviteEmails(t *testing.T) {
...
+		} else if len(resultsMailbox) > 0 {
+			require.Len(t, resultsMailbox, 1)
```
Comment (isacikgoz) [inline app/email/email_test.go:62]: I think we can remove the `len(resultsMailbox)` condition check if we opt to use `t.Skip` above as it's already going to stop if there are no resultsMailBox.

File: app/email/email_test.go [hunk 3]
```diff
@@ -40,34 +40,71 @@ func TestSendInviteEmails(t *testing.T) {
...
+		require.NoError(t, err)
 
-	var resultsMailbox mail.JSONMessageHeaderInbucket
-	err2 := mail.RetryInbucket(5, func() error {
-		var err error
-		resultsMailbox, err = mail.GetMailBox(emailTo)
-		return err
+		var resultsMailbox mail.JSONMessageHeaderInbucket
+		err2 := mail.RetryInbucket(5, func() error {
+			var err error
```
Comment (isacikgoz) [inline app/email/email_test.go:91]: What would you think about abstracting these assertions into a function, they seem to be used several times.

**Review by isacikgoz — COMMENTED** [review]
Nice catch! I have some cosmetic suggestions, otherwise good to go.

File: app/email/email_test.go [hunk 4]
```diff
@@ -38,29 +38,20 @@ func TestSendInviteEmails(t *testing.T) {
...
-		*cfg.ServiceSettings.EnableEmailInvitations = true
-	})
-
-	require.NotNil(t, th.BasicUser)
-	require.NotNil(t, th.BasicChannel)
-
 	emailTo := "test@example.com"
-	mail.DeleteMailBox(emailTo)
+	verifyMailbox := func(t *testing.T) {
+		t.Helper()
```
Comment (isacikgoz) [inline app/email/email_test.go:43]: 💯

**Review by isacikgoz — APPROVED** [review]
You are on fire man!

**Comment by agnivade** [comment]
https://github.com/mattermost/mattermost-server/pull/18299

**Pull request merged**